nototools/add_vs_cmap.py

```diff
@@ -56,7 +57,7 @@ def modify_font(font_name, font, presentation, emoji_variants):
 
 
 def modify_fonts(font_names, presentation='emoji', output=None, suffix=None,
-                 dst_dir=None):
+                 dst_dir=None, vs_added=None):
```

Comment:
variation selector.

this is how the entire tool is named, are you suggesting to change that too?